src/packet/ipv6.rs

```diff
@@ -22,7 +22,7 @@ pub struct Ipv6 {
     flow_label: u20be,
     payload_length: u16be,
     #[construct_with(u8)]
-    next_header: IpNextHeaderProtocol,
+    next_level_protocol: IpNextHeaderProtocol,
```

Comment:
It was renamed so that ipv4 and ipv6 will have the same method name for getting the IpNextHeaderProtocol... However it doesn't have to be done this way. Instead we could abstract this away in the implementation of the Pseudo Header trait.